classes/Player.py

```diff
@@ -12,6 +13,8 @@ def __init__(self,groups):
         self.can_shoot = True
         self.shoot_time= None
         self.bullet_sound=pygame.mixer.Sound('../sounds/laser.ogg')
+        self.font = pygame.font.Font('../graphics/subatomic.ttf',60)
+        self.game_over= False
```

Comment:
This path is being changed between commits. I believe it's because of the sublime and visual code studio. 
As long as it works locally, Ignoring this change.